raiden-dapp/tests/unit/components/transfer/transfer-menus.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { mount, Wrapper } from '@vue/test-utils';
+import Vue from 'vue';
+import Vuetify from 'vuetify';
+import TransferMenus from '@/components/transfer/TransferMenus.vue';
+import { BigNumber } from 'ethers/utils';
+import { One, Zero } from 'ethers/constants';
+import { generateToken } from '../../utils/data-generator';
+
+Vue.use(Vuetify);
+
+describe('TransferMenus.vue', () => {
+  const vuetify = new Vuetify();
+  const token = generateToken();
+
+  const createWrapper = (capacity: BigNumber): Wrapper<TransferMenus> => {
+    return mount(TransferMenus, {
+      vuetify,
+      stubs: ['v-menu'],
+      mocks: {
+        $t: (msg: string) => msg,
+      },
+      propsData: {
+        token,
+        capacity,
+      },
+    });
+  };
+
+  test('displays "no open channels" if channel capacity is zero', () => {
+    const wrapper = createWrapper(Zero);
+    const amountDisplay = wrapper.findAll('span').at(3);
```

Comment:
Nitpick: Here this only occurs 2x two times. But eventually you can export how to get the amount display and the deposit button into getter like functions. So you don't need to repeat yourself and the tests reads easy. Don't get me wrong. These test cases are pretty much perfect :ok_hand: These are single lines with well descriptive variable names which explain the not obvious selector. Here it is only a consolidation of a good habit and avoiding to replicated code. Just as a hint (I haven't seen the other commits with more tests yet...).

Reply:
Hmm, the test file won't let me define getter functions?